index.html

```diff
@@ -88,6 +88,20 @@ <h2 id="rules-wd">
         unnecessary and can potentially lead to unintended consequences.
       </p>
     </section>
+    <section class="informative">
+      <h2>ARIA semantics that extend/diverge from HTML</h2>
+      <p>Through the use of ARIA, authors can express semantics that go beyond the current native
+        HTML capabilities. As a result, there are situations where certain `aria-*` attributes are
+        allowed for use on elements with particular `role`s, while the equivalent native attribute
+        is currently not allowed in HTML itself.</p>
+      <p>For instance, HTML has no direct concept of a disabled link. Constructs such as `&lt;a href="..." disabled&gt; ... &lt;/a&gt;` are not valid, and will not be conveyed to assistive technologies. However, using ARIA, it is valid to
+        define a disabled link using `&lt;a href="..." aria-disabled="true"&gt; ... &lt;/a&gt;`. While user agents
```

Comment:
Just wondering... is there an issue filed for this in HTML? Seems kinda nice to do:

```
<a href="" disabled>
```

As an equivalent to using `aria-disabled="true"`

Reply:
this has been discussed... it really is just bad authoring and, imo, we should not be advocating for bad authoring.

Reply:
Wait, aria-disabled="true" is bad? I'm confused?

Reply:
No, I meant trying to make a link disabled specifically.

Reply:
Ah, got it. Yes, though I imagine people do it. Sorry, another silly question: how come it’s valid with aria-? Shouldn’t it also be invalid?

Reply:
i wouldn't be adverse to seeing `disabled` on links in HTML spec as well. i mean the "correct" current practice is to turn the link into another element or at the very least removing the `href` to make it a placeholder/anchor link. i could imagine spec'ing that if a link has a native `disabled` attribute that the browser should treat it as such / ignore the `href` and neutralise the default action for it